web/src/components/views/ProjectContainer.jsx

```diff
@@ -1,70 +1,105 @@
 import React, { Component } from 'react';
+import {fetchProjectDetails, fetchJson} from '../common/Backend'
 
-class ProjectContainer extends Component {
+export default class ProjectContainer extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      state:null
+      projectInf: []
     };
   }
 
+  componentWillReceiveProps(nextProps){
+    this.loadProjectInf(nextProps)
+  }
+
+  componentDidMount(){
+    this.loadProjectInf(this.props);
+  }
+
+  loadProjectInf(props) {
+    fetchProjectDetails(props.uuid).then(data => {
+      console.log(data);
```

Comment:
console logs are useful for debugging but not wanted in the final application. keep in mind to remove them later on ,-)